### ISSUE TITLE
Installed, Enabled, and Deployed mods have no effect on Skyrim

### ISSUE THREAD

**Issue by joezamoey8** [pikdum/steam-deck#84]
I just downloaded a large mod collection using pikdum's vortex app. The mod files show up in the data folder of Skyrim SE, but when I launch Skyrim in game mode, it's as if the mods aren't even there. The game runs like vanilla skyrim, without errors or crashes. I'm not sure what's going on here; but I successfully downloaded the mods from nexus using the Vortex download button.

 I then made sure that all the mods were enabled, clicked the deploy button, and double clicked the Skyrim AE/SE post deploy icon on the desktop. On running the game, nothing out of the ordinary happened. In an attempt to get the mods running, I set the staging folder for the mods to the same location as the game (J:\vortex_mods\{GAME}) and replaced the SKSELauncher with the skse64loader by renaming the original launcher _SKSELauncher.exe and then renaming the skse64loader to SKSELauncher.exe. I've also set the game to run with GE-Proton9-2.

One issue i have is with the game version. I replaced the current version of skyrim(1.6.112) by writing in files downloaded from these depot codes:
download_depot 489830 489831 3660787314279169352

download_depot 489830 489832 2756691988703496654

download_depot 489830 489833 5291801952219815735

Under vortex, the installed version of Skyrim is listed as 1.6.640.0; however, when I check in game by using console code getSKSEVersion, I get a message saying "Script command "getSKSEVersion" not found." on checking at the bottom left corner of the system page in the journal, using an old save that I made before I wrote the depot files in, the version is listed as 1.6.1130.0.8. 

Under the voretex bell, I get these notifications:

Exception
Last Vortex session crashed
*screen froze when I went to select the file location for skyrim, which is not listed

Missing Masters
*for Convenient horses, which depends on plugins which aren't enabled(I've checked and enabled all plugins)

Failed to Update Master List
*I can't find the file that needs to be deleted(masterlist) on the steam deck, since the paths in which files are stored are different from the pc

Checking for update failed
*because I opened vortex in offline mode

Failed to get collection Info 
*Listed as most likely being a temporary issue

What do I need to do to get these mods to work?

**Comment by pikdum** [comment]
My best guess is that your loadorder.txt and/or plugins.txt aren't being copied over properly, or maybe there are multiple with different casing.

**Comment by joezamoey8** [comment]
After reinstalling Pikdum's vortex launcher, I received a message saying that the Skyrim ini file was not detected, even though I had previously launched the game multiple times with the new game option, waiting until I saw Ralof talking.

After reinstalling Skyrim and immediately writing in the version I had acquired through download depot commands on the steam launcher console, the version I had intended to install was listed in the game's journal. 

After downloading the mod collection through a new user, the game is running well with working mods.
